assets/svg/wallet_icon_wallet_report.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this correct size of SVG icon?

Reply:
Rechecked the icon, looks like this size is correct.